internal/featuretests/v3/secrets_test.go

```diff
@@ -171,7 +171,7 @@ func TestSDSShouldNotIncrementVersionNumberForUnrelatedSecret(t *testing.T) {
 
 // issue 1169, an invalid certificate should not be
 // presented by SDS even if referenced by an ingress object.
-func TestSDSshouldNotPublishInvalidSecret(t *testing.T) {
+func TestSDSShouldNotPublishInvalidSecret(t *testing.T) {
```

Comment:
It's certainly fine to drop this, but a good tool/human will be able to properly parse this as:

Test SDS Should Not Publish Invalid Secret

Just as they should be able to properly parse HTTPProxy as HTTP Proxy (as long as it's properly spelled...) or TestUpstreamTLSWithHTTPRoute...